ingest/workflow/snakemake_rules/nextclade.smk

```diff
@@ -74,13 +74,13 @@ rule join_metadata_clades:
 
         export APPEND_FIELDS=`awk 'NR>1 {{print $2}}' {input.nextclade_field_map} | grep -v -e "index" -e "seqName" | tr '\n' ',' | sed 's/,\$//g'`
 
-        tsv-join -H \
+        cat {input.metadata} | tr '\r' '\n' | grep -v "^$" \
```

Comment:
I opted for the `lineterminator='\n'` route https://github.com/nextstrain/monkeypox/pull/207/commits/233457ababe49468b39ff4797668dd2ac197be83

While I acknowledge that the `augur curate passthru` route would be more elegant, the implementation ended up resembling the following:

```
|  augur curate passthru \
  --output-metadata - \
  --output-fasta data/sequences.fasta \
  --output-id-field accession \
  --output-seq-field sequence \
| tsv-select -H \
  --fields accession,genbank_accession_rev,strain,date,region,country,division,location,host,date_submitted,sra_accession,abbr_authors,reverse,authors,institution  \
  > data/metadata_raw.tsv
```

Perhaps we could consider introducing an `--output-metadata-fields` flag, particularly for fields that are absent (e.g. non-existent column `reverse` causes `tsv-select` to error out).

Reply:
(100% fine to leave as-is, just commenting for posterity)

Ah, I specifically didn't feel the need to add an `--output-metadata-fields` option because you can chain it with `tsv-select` to subset the columns 😅 

As for the `reverse` column, it seems like we can completely remove this column from the metadata output since Nextclade includes an `is_reverse_complement` column. The phylogenetic builds now use the `is_reverse_complement` column since https://github.com/nextstrain/monkeypox/pull/191. Maybe @corneliusroemer can help confirm here that the `reverse` column is no longer needed.

Reply:
Moving discussion of `reverse` column to a [new issue](https://github.com/nextstrain/monkeypox/issues/209).